src/types/asam/index.ts

```diff
@@ -29,15 +31,13 @@ import { Plugin, Cordova, IonicNativePlugin } from '@ionic-native/core';
 })
 @Injectable()
 export class ASAM extends IonicNativePlugin {
-
   /**
    * Tries to toggle ASAM.
    * @param flag {boolean} true enables ASAM, false disable ASAM
    * @return {Promise<boolean>} Returns a promise that resolves with boolean value that describes if toggle succeeded
    */
   @Cordova()
-  toggle(flag: boolean): Promise<boolean> {
+  toggle(_flag: boolean): Promise<boolean> {
```

Comment:
What's the reason behind prepending the `_` to `flag`? Also, the JSDoc block needs to be updated with the underscore

Reply:
Agree, avoid underscores if we can.

Reply:
sorry guys, I added a note to PR summary but I thought that it can be written even clearer. 
So the _ is a way to mute tsc warnings. But going to the very beginning.
I noticed that VSC do not show warnings about unused properties. I took a look why is that and found out that since new (5) version of tslint is not supporting unused variable rule. (see FAQ https://marketplace.visualstudio.com/items?itemName=eg2.tslint) and the recommended workaround it turns **noUnusedLocals**, **noUnusedParameters** on in tsconfig.json

I did it, but now it started complaining (correctly!) that **flag** is not used. and the only way to mute it was added _ a prefix

Actually, there is another solution, move these types classes so separate repo and import them whenever we need...